applications/luci-app-cloudflared/root/usr/share/luci/menu.d/luci-app-cloudflared.json

```
@@ -0,0 +1,29 @@
+{
+	"admin/vpn/cloudflared": {
+		"title": "Cloudflared",
+		"order": 80,
+		"action": {
+			"type": "firstchild"
+		},
+		"depends": {
+			"acl": [ "luci-app-cloudflared" ],
+			"uci": { "cloudflared": true }
+		}
+	},
+	"admin/vpn/cloudflared/base": {
+		"title": "Base settings",
```

```suggestion
	"admin/vpn/cloudflared/config": {
		"title": "Configuration",
```

`Configuration` might be a more general title used in this repo.

---

changed